Mathlib/CategoryTheory/Core.lean

```diff
@@ -17,46 +17,46 @@ import Mathlib.CategoryTheory.Types
 # The core of a category
 
 The core of a category `C` is the (non-full) subcategory of `C` consisting of all objects,
-and all isomorphisms. We construct it as a `groupoid`.
+and all isomorphisms. We construct it as a `Groupoid`.
 
-`core.inclusion : core C ⥤ C` gives the faithful inclusion into the original category.
+`Core.inclusion : Core C ⥤ C` gives the faithful inclusion into the original category.
 
-Any functor `F` from a groupoid `G` into `C` factors through `core C`,
+Any functor `F` from a groupoid `G` into `C` factors through `Core C`,
 but this is not functorial with respect to `F`.
 -/
 
-
 namespace CategoryTheory
 
 universe v₁ v₂ u₁ u₂
 
--- morphism levels before object levels. See note [category_theory universes].
+-- morphism levels before object levels. See note [CategoryTheory universes].
```

Comment:
The note is called `category_theory universes`. If you want to rename it, then please open a separate PR.